pennylane_forest/wavefunction.py

```diff
@@ -95,17 +96,19 @@ def expand_state(self):
             # all wires in the device have been initialised
             return
 
-        # there are some wires in the device that have not yet been initialised
-        inactive_wires = set(range(self.num_wires)) - self._active_wires
+        num_inactive_wires = len(self.wires) - len(self._active_wires)
```

Comment:
We only need the number of inactive wires here...and the `Wires` objects make sure that labels are unique!

Reply:
Nice!